tools/shared.py

```diff
@@ -320,10 +320,14 @@ def fix_js_engine(old, new):
 
 # Expectations
 
-EXPECTED_LLVM_VERSION = (4, 0)
-
 actual_clang_version = None
 
+def expected_llvm_version():
+  if get_llvm_target() == WASM_TARGET:
```

Comment:
I see yes. We have to give the warning to someone, and right now that is the tip-of-tree users.   Make more sense to warn if user tries to use wasm backend with fastcomp which is a far less tested path right?